src/backend.rs

```diff
@@ -80,11 +76,15 @@ impl MySqlBackend {
 
     fn do_insert(&mut self, table: &str, vals: Vec<Value>, replace: bool) {
         let op = if replace { "REPLACE" } else { "INSERT" };
+        //changes to be made here
+        let mut insert_vals = String::new(); 
+        let temp = vals.iter().map(|_| "?").collect::<Vec<&str>>().join(",");
+        insert_vals += &temp;
```

Comment:
Seems like it just creates an empty string, concatenates the question marks using the same code removed from below and then appends to the empty string. Why is it needed -- the resulting string seems to be the same as the one that the code you removed below would have generated?

Reply:
I originally made these changes as I modified this method to make all necessary changes to inserts. I later changed that as you told me to but accidentally left this as is. Changing it now!